readme.md

```diff
@@ -59,6 +59,7 @@
 - [Acho Arnold](https://github.com/najela/ama) - Full Stack Web Developer with great passion in [Social Psychology](https://en.wikipedia.org/wiki/Social_psychology).
 - [Rob Furlong](https://github.com/robbbz/ama) - Maker of cool things with the web, freelancer.
 - [Honza Bittner](https://github.com/HoBi/ama) - High school student, web developer, Sass and Polymer ❤️'r.
+- [Edenspiekermann](https://github.com/edenspiekermann/ama) - Design agency based in Berlin and around the world.
```

Comment:
I don't see why we should limit it to people. Many of the best AMAs on Reddit are from companies. It definitely should be a separate category though.